spread/project.go

```diff
@@ -39,7 +39,8 @@ type Project struct {
 	Exclude []string
 	Rename  []string
 
-	Path string `yaml:"-"`
+	Path   string `yaml:"-"`
+	Subdir string `yaml:"project-subdir"`
```

Comment:
project-dir?  Or even project-root?

Do we have any other analogous options at the moment?

The "subdir" bit feels a bit off considering that this is the topmost directory we have.

Reply:
The existing similar case is `source-subir` in parts, but I'll be happy with `project-root` here.

Reply:
As discussed, we have `path` already which represents the remote path, so we need to think about how to align both the terms used as well as to make the distinction between the local copy and the remote copy more obvious.

Reply:
Per discussion, let's try `reroot` to see if it works well, in the same family of `repack` and `rename`.

Reply:
Updated the attribute name to `reroot` and added a usage note to the documentation.